# Robot left and right motor command


uint16 left_speed
uint16 right_speed